pom.xml

```diff
@@ -92,6 +92,10 @@
                             <pattern>javassist</pattern>
                             <shadedPattern>net.sacredlabyrinth.phaed.simpleclans.javassist</shadedPattern>
                         </relocation>
+                        <relocation>
+                            <pattern>discordsrv</pattern>
+                            <shadedPattern>net.sacredlabyrinth.phaed.simpleclans.discordsrv</shadedPattern>
```

Comment:
Is it necessary to relocate DiscordSRV? DiscordSRV is a plugin, so it loads itself.